pkg/render/apiserver.go

```diff
@@ -336,6 +334,9 @@ func (c *apiServerComponent) Objects() ([]client.Object, []client.Object) {
 	// Compile the final arrays based on the variant.
 	if c.cfg.Installation.Variant == operatorv1.TigeraSecureEnterprise {
 		// Create any enterprise specific objects.
+		globalObjects = append(globalObjects,
+			CreateNamespace(rmeta.APIServerNamespace(operatorv1.TigeraSecureEnterprise), c.cfg.Installation.KubernetesProvider, PSSPrivileged, c.cfg.Installation.Azure),
```

Comment:
I don't think we need create namespace here at all any more, right? For enterprise, this is in calico-system, which will have been created already by the core controller?